theme/templates/accounts/profile.html

```diff
@@ -1029,6 +1032,14 @@ <h4 class="modal-title" id="deleteCV">Delete CV</h4>
 
             $("tr[data-type='all']").find(".badge").text(collection["total"]);
 
+
+            // Unspecified goes away as soon as a user clicks.
+            $("input[name='state'").click(function () {
```

Comment:
@mauriel133 
I think there is a typo. this line missed a right square bracket ] after 'state'.